### ISSUE TITLE
Browser runner fails loading tests

### ISSUE THREAD

**Issue by christian-bromann** [vitest-dev/vitest#3045]
### Describe the bug

When running a basic browser runner test, it fails as some requests stay pending or fail:

![Screenshot 2023-03-20 at 12 18 14](https://user-images.githubusercontent.com/731337/226443565-58b02aa4-0a8a-4bc7-a7c4-a339c32c2418.png)

### Reproduction

Create an empty dir and setup a Vitest projects using:

__vitest.conf.js__
```js
export default defineConfig({
  test: {
    browser: {
      enabled: true,
      name: 'chrome',
    },
  }
})
```

a simple test at `/tests/browser.test.js`:
```js
import { test } from 'vitest'

test('I live in the browser', () => {
    console.log('Hello World', window.navigator.userAgent)
})
```

### System Info

```shell
System:
    OS: macOS 12.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 953.73 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 18.7.0 - ~/.nvm/versions/node/v18.7.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v18.7.0/bin/yarn
    npm: 8.15.0 - ~/.nvm/versions/node/v18.7.0/bin/npm
  Browsers:
    Brave Browser: 111.1.49.120
    Chrome: 111.0.5563.64
    Firefox: 109.0
    Firefox Nightly: 111.0a1
    Safari: 15.2
  npmPackages:
    @vitest/browser: ^0.29.5 => 0.29.5
    vitest: ^0.29.5 => 0.29.5
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
For `__vitest__`, it seems UI did not get bundled for some reason 🤔 

As for the second one, it seems Vite optimizer is trying to optimize Vitest and doesn't finish for some reason 🤔

**Comment by sheremet-va** [comment]
Should be fixed in [v0.29.7](https://github.com/vitest-dev/vitest/releases/tag/v0.29.7). Thanks for the report 🙏🏻

**Comment by christian-bromann** [comment]
Thanks for the quick fix.
